packages/twenty-front/src/utils/date-utils.ts

```diff
@@ -117,12 +119,15 @@ export const beautifyDateDiff = (
   date: string,
   dateToCompareWith?: string,
   short = false,
+  infiniteDiffTriggerInYears = NEVER_EXPIRE_DELTA_IN_YEARS / 10,
```

Comment:
Are we displaying +infinite somewhere? If it's only to use it in the condition mentionned above, it seems to be too hacky